bin/OpenDataDirectory.bat

```diff
@@ -0,0 +1 @@
+start %LocalAppData%\ClanGen\ClanGen
```

Comment:
n o g o d n o

Reply:
Yes. Say "thank you" to Windows for requiring elevation to create symlinks and for having an _awful_ API for creating shortcuts.

Reply:
i don wanna >:(